shared/react-native/android/app/src/main/AndroidManifest.xml

```diff
@@ -64,7 +64,7 @@
       </service>
       <provider
           android:name="io.keybase.ossifrage.util.SharedFileProvider"
```

Comment:
Does this need to change as well?

Also, would it work for test build that we are still importing the non test version here? https://github.com/keybase/client/pull/11428/files#diff-f359894605123aad5332fc343c11563cR34

Reply:
No, this part can't change because it's in the class structure.

Reply:
And re: the second question, weird, it's still working for me.